Runigram.java

```diff
@@ -37,13 +43,17 @@ public static Color[][] read(String fileName) {
 		in.readInt();
 		// Creates the image array
 		Color[][] image = new Color[numRows][numCols];
-		// Reads the RGB values from the file, into the image array. 
-		// For each pixel (i,j), reads 3 values from the file,
-		// creates from the 3 colors a new Color object, and 
-		// makes pixel (i,j) refer to that object.
-		//// Replace the following statement with your code.
-		return null;
-	}
+		for(int i=0; i<numRows; i++)
+		{
+			for(int j =0; j< numCols; j++)
+			{	
+				image[i][j] = new Color(in.readInt(),in.readInt(),in.readInt());

```

Comment:
That's way it much more clear what are you expecting to get here.
```suggestion
				int red = in.readInt();
				int blue = in.readInt();
				int green = in.readInt();
				image[i][j] = new Color(red, blue, green);

```